tin/settings/markdown.py

```diff
@@ -0,0 +1,14 @@
+from pygments.formatters import HtmlFormatter
+
+
+class HtmlCodeFormatter(HtmlFormatter):
+    def __init__(self, lang_str: str = "", **kwargs):
+        super().__init__(**kwargs)
+        # lang_str has the value {lang_prefix}{lang}
+        # specified by the CodeHilite's options
+        self.lang_str = lang_str
+
+    def _wrap_code(self, source):
+        yield 0, f'<code class="{self.lang_str}">'
+        yield from source
+        yield 0, "</code>"
```

Comment:
I don't think this custom formatter is necessary.

With the formatter: `<code class="language-py hljs language-python">`
Without the formatter: `<code class="hljs language-python">`

Both versions seem to be styled properly with `hljs`...